compiler/src/dotty/tools/dotc/reporting/messages.scala

```diff
@@ -2743,8 +2744,15 @@ class MissingImplicitArgument(
         // show all available additional info
         def hiddenImplicitNote(s: SearchSuccess) =
           i"\n\nNote: ${s.ref.symbol.showLocated} was not considered because it was not imported with `import given`."
+        def noChainConversionsNote(s: Iterable[TermRef]): Option[String] =
+          Option.when(s.nonEmpty)(
+              i"\n\nNote: Chaining implicit conversions are not allowed in Scala. " +
```

Comment:
As I noted before, this is not only about chaining conversions so this message might be misleading. I would suggest something like this instead:
```
Note: implicit conversions are not automatically applied to arguments of using clauses
```